namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-manage-adjudications-api-preprod/resources/rds.tf

```diff
@@ -37,3 +37,19 @@ resource "kubernetes_secret" "dps_rds" {
   }
 }
 
+# This places a secret for this preprod RDS instance in the production namespace,
+# this can then be used by a kubernetes job which will refresh the preprod data.
+resource "kubernetes_secret" "dps_rds_refresh_creds" {
+  metadata {
+    name      = "dps-rds-instance-output-preprod"
+    namespace = "hmpps-manage-adjudications-api-prod"
+  }
+
+  data = {
+    rds_instance_endpoint = module.dps_rds.rds_instance_endpoint
```

Comment:
Should this be `module.ma_rds.`

Reply:
yes - it should sorry about that - fixed